consumer-sniffer/src/main/kotlin/com/zynger/floorplan/FloorPlanConsumerSniffer.kt

```diff
@@ -7,16 +7,12 @@ import java.io.File
 
 object FloorPlanConsumerSniffer {
 
-    fun sniff(inputSourceFile: File): Consumer {
-        return when (inputSourceFile.extension) {
-            "json" -> RoomConsumer
-            "db" -> SqliteConsumer
-            "dbml" -> DbmlConsumer
-            else -> throw IllegalArgumentException("Unknown file extension: ${inputSourceFile.extension}")
-        }
-    }
+    private val extensionToConsumer = mapOf("json" to RoomConsumer, "db" to SqliteConsumer, "dbml" to DbmlConsumer)
 
-    fun isConsumable(inputSourceFile: File): Boolean {
-        return inputSourceFile.extension == "json" || inputSourceFile.extension == "db"
-    }
+    fun sniff(inputSourceFile: File): Consumer =
+        if (isConsumable(inputSourceFile))
+            extensionToConsumer.getValue(inputSourceFile.extension)
+        else throw IllegalArgumentException("Unknown file extension: ${inputSourceFile.extension}")
```

Comment:
please add braces to the `if-else` construct

Reply:
Done.